eva-lib/src/test/java/uk/ac/ebi/eva/lib/metadata/ArchiveEvaproDBAdaptorTest.java

```diff
@@ -25,6 +25,7 @@
 
 import uk.ac.ebi.eva.lib.entities.Project;
 import uk.ac.ebi.eva.lib.entities.Taxonomy;
+import uk.ac.ebi.eva.lib.metadata.eva.ArchiveEvaproDBAdaptor;
```

Comment:
This test should be moved to the same package that the class that is testing:  uk.ac.ebi.eva.lib.metadata.eva